.golangci.yml

```diff
@@ -48,6 +48,7 @@ linters-settings:
             - github.com/pachyderm/pachyderm/v2/src/internal/errors.Invoke1
             - google.golang.org/grpc/status.Error
             - google.golang.org/grpc/status.Errorf
+            - "(*google.golang.org/grpc/internal/status.Status).Err"
```

Comment:
why is this necessary?

Reply:
Allows getting rid of `//nolint:wrapcheck` when returning a gRPC status error.

Not strictly necessary in this PR, but it’s about to be used in the follow-ons.